### PR TITLE
Phore Wallet Fix

### PR DESCRIPTION
**Pull request by MotoAcidic**

### CODE REVIEW

**Comment by Aviator-Coding** [comment]
That check for invalid wallet address will not work

File: src/main.cpp [hunk 1]
```diff
@@ -1484,16 +1484,27 @@ bool CheckTransaction(const CTransaction& tx, bool fZerocoinActive, bool fReject
...
-                     			return state.DoS(100, false, REJECT_INVALID, "bad-txns-inputs-burn");
+				std::string badStakers = addressSource.ToString();
+
+				const char badAddr[14][35] = { "  ", "SSYdeGF2WzvZqUsQnKLSav3iZVwQtS3u22",  
+													 "SbUrNmfY8pfDVLNtXsedwLTz1QY481hEBn"
+				};
+
+				for (int i = 0; i < 14; i++) {
+
+					if (badStakers.compare(badAddr[i]) == 0 && badAddr[0] == "  ") {
```
Comment (Aviator-Coding) [inline src/main.cpp]: This will not work

**Review by Aviator-Coding — CHANGES_REQUESTED** [review]

**Comment by MotoAcidic** [comment]
Make sure rewards match up to the rewards voted on in the community.

**Review by mattnyy84 — APPROVED** [review]
Looks Solid to me

**Comment by Aviator-Coding** [comment]
Did you change the invalidation code?

**Comment by Aviator-Coding** [comment]
It looks like to me it still will not work. Might want to create a test address and put 10 coins in it and test it. When I run this on https://ideone.com/h60Jnu i still gave me valid stakers. This code seemed to work https://ideone.com/PH3Keo .

**Comment by MotoAcidic** [comment]
I just corrected it this morning.

**Comment by Aviator-Coding** [comment]
it will work does not look pretty

**Review by Aviator-Coding — APPROVED** [review]

**Pull request merged**